### DISCUSSION TITLE
Multiple Search Filters web parts connected to same search results web part

### DISCUSSION THREAD

**Discussion by tvakisai** [microsoft-search/pnp-modern-search#3161] [Q&A] [answered]
Is there a way to connect two search filters web parts to the same search results? The appears to be a limitation as you can only selected 1 search filters web part on the search results web part configuration.

**Comment by FranckyC** [comment, marked as answer]
Hi @tvakisai, this setup is not supported unfortunately. You can only connect one search filters from a search results. However, you can connect multiple search results from a search filter WebPart in order to support scenarios with verticals. Can you describe your use case?

**Reply by tvakisai** [reply]
Hi @FranckyC , thank you for your the quick response. I have eight (8) properties I would like to use as filters. It is a lot to fit on one filters web part. I would like to have the first four properties on a search filters web part with a horizontal layout on a full-width section. The  other four properties will be on the left section below that with a vertical layout. The search results will be on the right section. Hope this helps?

**Reply by FranckyC** [reply]
You could achieve that using a custom template, custom logic and CSS on the filters WebPart. Did you also try the panel layout? That may be a solution to avoid to avoid many filters displayed at the page load.

**Reply by tvakisai** [reply]
Thanks @FranckyC . I will explore the custom template route as we need all the filters displayed on the page.
